stable/artifactory/values.yaml

```diff
@@ -98,7 +98,6 @@ nginx:
   replicaCount: 1
   image:
     repository: "docker.bintray.io/jfrog/nginx-artifactory-pro"
-    version: 5.6.3
```

Comment:
What is someone wanted to override the nginx or artifactory images? They would have change any version tags as a group rather than one or the other.

And, if someone reads the values.yaml file as a consumer of the chart they may be confused on setting the version.

Could you please revert where the versions are combined?